### PR TITLE
CCM3-101 adding Openshit build info to UI

### PR DESCRIPTION
**Pull request by pushyamig**
https://itsjira.umms.med.umich.edu/browse/CCM3-101

### CODE REVIEW

**Comment by pushyamig** [comment]
[CCM3-101 adding Openshift info to the UI](https://trello.com/c/uayzwPdA/1253-ccm3-101-adding-openshift-info-to-the-ui)

File: src/main/webapp/index-lti.vm [hunk 1]
```diff
@@ -103,7 +103,17 @@
         <div ng-include src="'assets-lti/views/friends.html'"></div>
       </div>
     </div>
-
+    <a href="" style="position:fixed;right:0;top:60px;width:30px" ng-click="showDebug = !showDebug">&nbsp;</a>
```
Comment (gsilver) [inline src/main/webapp/index-lti.vm]: add aria-hidden="true" attribute to link, might as well add it to the div too.

File: src/main/webapp/index-sc.vm [hunk 2]
```diff
@@ -241,6 +241,17 @@
         <div ng-include src="'assets-sc/views/friends.html'"></div>
       </div>
     </div>
+    <a href="" style="position:fixed;right:0;top:60px;width:30px" ng-click="showDebug = !showDebug">&nbsp;</a>
```
Comment (gsilver) [inline src/main/webapp/index-sc.vm]: add aria-hidden"true" to this as well.

File: src/main/webapp/index-lti.vm (hunk 1 above)
Comment (pushyamig) [reply src/main/webapp/index-lti.vm]: Done

File: src/main/webapp/index-sc.vm (hunk 2 above)
Comment (pushyamig) [reply src/main/webapp/index-sc.vm]: Done

**Pull request merged**